### DISCUSSION TITLE
Variable from detected object

### DISCUSSION THREAD

**Discussion by domcio01** [AlexeyAB/darknet#8485] [Q&A]
Hi :) i run darknet with command : ./darknet detector demo ....weights
and i have no clue where should i search for a code that recognise objects
I'd like this darknet to save in variable last thing which was recognized eg.
if it recognizes dog makes new variable a =1 , if it recognizes human makes new variable a=2 ... e.t.c
which file is responsible for recognising?
is there any chance to pick thing with biggest precision?

**Comment by stephanecharette** [comment]
Use the API instead of the command-line option.  Then you can make it do whatever you want.

**Reply by domcio01** [reply]
sure ill try this but do you have knowledge which file is responsible for running it ?
